client/my-sites/marketplace/components/reviews-modal/styles.scss

```diff
@@ -1,45 +1,78 @@
-@import "@automattic/typography/styles/variables.scss";
+$border-color: #eee;
 
-.marketplace-reviews-modal__card {
-	padding: 0 1rem;
-	box-shadow: none;
-}
+.marketplace-reviews-modal {
+	max-width: 850px;
```

Comment:
If there's no long reviews for a plugin yet (eg. with `/plugins/woocommerce-subscriptions`), the modal looks a bit cramped, especially with the wide CTA:
![CleanShot 2023-12-20 at 13 11 00@2x](https://github.com/Automattic/wp-calypso/assets/11555574/5f6e7f2a-3b3d-4d51-a89c-4deb76caf455)

Maybe we should add a `min-width` here too to compensate this.

Reply:
I've added a few intermediary steps here, let me know how that works for you: https://github.com/Automattic/wp-calypso/pull/85496/commits/4b7e00298c74c2585554457bba79e74a8bf9580b

Reply:
Also added a gap to avoid them to get too close https://github.com/Automattic/wp-calypso/pull/85496/commits/135c84af3865e0bb3a5964438dd86ae2b30c81f7

Reply:
Brilliant, thanks so much for iterating on this!

Reply:
It looks great now on all breakpoints 👍 
![CleanShot 2023-12-20 at 15 57 42@2x](https://github.com/Automattic/wp-calypso/assets/11555574/b93701d7-fc2a-4205-a8bb-eeddd5c43601)

`<960`:
![CleanShot 2023-12-20 at 15 58 17@2x](https://github.com/Automattic/wp-calypso/assets/11555574/4ad0b45c-7334-49f3-9f5f-11c42ce27d63)

`<660`:
![CleanShot 2023-12-20 at 15 58 23@2x](https://github.com/Automattic/wp-calypso/assets/11555574/3b742c18-19b4-40be-a14c-798c5650152b)